src/Extensions/DoctrineConnectionDynamicReturnTypeExtension.php

```diff
@@ -4,6 +4,8 @@
 
 namespace staabm\PHPStanDba\Extensions;
 
+use Composer\InstalledVersions;
+use Composer\Semver\VersionParser;
```

Comment:
You need to require composer/semver if you want to use `satisfies`.

Reply:
Thx for the hint. Interesting that the build is green even though the dependency is not declared

Reply:
Probably something else you depend on requires it, so it works but not super failsafe :)